popup.js

```diff
@@ -0,0 +1,79 @@
+// Get the modal
+const modal = document.getElementById('myModal');
+const modal2 = document.getElementById('myModal2');
+const modal3 = document.getElementById('myModal3');
+const modal4 = document.getElementById('myModal4');
+const modal5 = document.getElementById('myModal5');
+const modal6 = document.getElementById('myModal6');
+const modal7 = document.getElementById('myModal7');
```

Comment:
- [ ] Following the above comment, please keep only one modal, and change the informations based on project objects. As per requirement, you should create an array of objects containing data about each project, which you'll be using to fill the informations inside both the modal window and the project card itself. 
Example: 
```
const projects = [{
    name: "Project one",
    description: "Description one",
    image: "link_to_image.jpg",
    technologies: ["CSS", "JavaScript", "HTML"],
    link: "link_to_live_version.com",
}]

```

Reply:
resolved